packages/console/src/pages/Connectors/index.module.scss

```diff
@@ -22,13 +23,11 @@
   width: 360px;
 }
 
-a.link {
-  color: inherit;
-  text-decoration: none;
+.empty {
```

Comment:
Sounds like a general CSS functional style.
How about `socialCreateHintContainer` or something like that?

Reply:
css module will auto add a random prefix to it, so we don't need a redunctant "social" prefix, since is inside connector's css module.

Reply:
per offline discussion, `empty` is commonly used in <table> to represent the state when there are no data inside.